netlify.toml

```
@@ -3,9 +3,16 @@
     rustup install nightly --profile minimal \
       && cargo doc --no-deps
     """
-  environment = { RUSTDOCFLAGS= "--cfg docsrs" }
   publish = "target/doc"
 
+[build.environment]
+  RUSTDOCFLAGS="""
+    --cfg docsrs \
+    --html-before-content $(pwd)/assets/warning.html \
+    --html-in-header $(pwd)/assets/noindex.html \
+    --extend-css $(pwd)/assets/warning.css
```

hmm, maybe the problem here is a trailing newline?
```suggestion
    --extend-css $(pwd)/assets/warning.css \
```